daml-script/export/src/main/scala/com/daml/script/export/Encode.scala

```diff
@@ -346,12 +350,14 @@ private[export] object Encode {
       cidMap: Map[ContractId, String],
       r: Record,
   ): Doc = {
-    if (r.fields.isEmpty) {
+    if (r.fields.isEmpty)
       qualifyId(r.getRecordId)
-    } else {
-      (qualifyId(r.getRecordId) + Doc.text(" with") /
-        Doc.stack(r.fields.map(f => encodeField(partyMap, cidMap, f)))).nested(2)
-    }
+    else
+      parens(
+        qualifyId(r.getRecordId) &
+          braces(Doc.intercalate(comma, r.fields.map(encodeField(partyMap, cidMap, _))))
+      )
```

Comment:
I don't think the outer parens here is necessary. Curly brace syntax binds tightly.